src/avatar/Avatar.tsx

```diff
@@ -15,87 +28,169 @@ export type AvatarProps = {
    */
   name?: string;
   /**
-   * How large should avatar be?
+   * Defines loading strategy
    */
-  size?: keyof Renderlesskit.GetThemeValue<"avatar", "size">;
+  loading?: "eager" | "lazy";
   /**
-   * Custom fallback
+   * Function to get the initials to display
    */
-  fallback?: React.ReactNode;
+  getInitials?: (name: string) => string;
```

Comment:
Make type

```
  getInitials?: (name?: string) => string | undefined;
```

to make it compatible with this change https://github.com/timelessco/renderlesskit-react-tailwind/pull/63/files#r575204267